### PR TITLE
Add more returndata tests

### PR DESCRIPTION
**Pull request by jwasinger**

### CODE REVIEW

File: src/GeneralStateTestsFiller/stReturnDataTest/returndatasize_after_failing_staticcallFiller.json [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+                "code": "{ (seq (STATICCALL 60000 0xa94f5374fce5edbc8e2a8697c15331677e6ebf0b 0 0 0 0) (SSTORE 0 (RETURNDATASIZE)))}",
+                "nonce" : "0x00",
+                "storage" : {
+                    "0x00" : "0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff"
+                }
+            },
+            "0xa94f5374fce5edbc8e2a8697c15331677e6ebf0b" : {
+                "balance" : "0x6400000000",
+								"//code": "an invalid opcode",
+                "code" : "0xfd",
```
Comment (winsvega) [inline src/GeneralStateTestsFiller/stReturnDataTest/returndatasize_after_failing_staticcallFiller.json]: it is not good to have sender with code. some clients check that this is impossible situation.

Comment (jwasinger) [reply src/GeneralStateTestsFiller/stReturnDataTest/returndatasize_after_failing_staticcallFiller.json]: Thanks for pointing that out.  You're referring to the distinction between 'simple' accounts and 'contract' accounts right?

It should be easy enough to modify the tests to accommodate for this.

File: GeneralStateTests/stReturnDataTest/returndatacopy_after_failing_callcode.json [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+            "0x0f572e5295c57f15886f9b263e2f6d2d6c7b5ec6" : {
+                "balance" : "0x10000000",
+                "code" : "",
+                "nonce" : "0x00",
+                "storage" : {
+                }
+            },
+            "0x1000000000000000000000000000000000000001" : {
+                "balance" : "0x00",
+                "code" : "{  (CALLCODE 0 0xa94f5374fce5edbc8e2a8697c15331677e6ebf0b 0 0 0 0 0) (RETURNDATACOPY 0x0 0x0 32) (SSTORE 0 (MLOAD 0))}",
```
Comment (winsvega) [inline GeneralStateTests/stReturnDataTest/returndatacopy_after_failing_callcode.json]: for some reason this test has not been filled. it just a copy of the filler in destenation test folder

**Comment by pirapira** [comment]
I marked the cells green. https://docs.google.com/spreadsheets/d/1xat7UI8GtB4ZGVdlK5_XQSHJZaMThi4SrlcL8XMZb5Q/edit#gid=1605675414

**Pull request merged**